src/components/SubmitResponsePage.jsx

```diff
@@ -1,87 +1,96 @@
 import React from 'react';
+import Alert from './Alert';
 import FormContainer from './FormContainer';
 import { Formik, Form } from 'formik';
 import { HasResponseErrors } from '../utilities/CitysourcedResponseHelpers';
-import ButtonDisplay from "./buttonDisplay";
+import ButtonDisplay from './buttonDisplay';
+
+const successBodyContent = (
+	<React.Fragment>
+		<p>
+			Thank you for submitting your report. You will receive an email in a few minutes with your tracking number
+			and additional information.
+		</p>
+		<p>
+			You can track your status online at any time by entering your tracking number at{' '}
+			<a href="/followup" title="Track your issue status online.">
+				www.baltimorecountymd.gov/followup
+			</a>.
+		</p>
+	</React.Fragment>
+);
+
+const failureBodyContent = (
+	<React.Fragment>
+		<p>
+			We're sorry, we encountered a problem processing your submission. We are working to resolve this issue as
+			quickly as possible.
+		</p>
+	</React.Fragment>
+);
+
+const getAlertInfo = (title, bodyContent, type, controls) => ({
+	title,
+	bodyContent,
+	type,
+	controls
+});
 
 const SubmitResponse = (props) => {
-	const {
-		Tabs,
-		shouldDisableForm,
-		requiresLocation
-	} = props.values;
-	const {
-		response = {}
-	} = props.history.location.state || {};
+	const { Tabs, shouldDisableForm, requiresLocation } = props.values;
+	const { response = {} } = props.history.location.state || {};
 	const isFormSubmissionSuccessful = !HasResponseErrors(response);
-
 	const resetForm = () => {
 		props.resetForm();
 		props.history.push('/ServiceRequestForm');
-	}
-
-	const returnHome = () =>{
+	};
+	const returnHome = () => {
 		resetForm();
-	}
-
+	};
 	const logout = () => {
 		sessionStorage.clear();
 		resetForm();
-	}
+	};
+	const HomeButton = (
+		<React.Fragment>
+			<ButtonDisplay key={1} onClick={returnHome} buttonName="Create New Report" cssClass="seButton" />
+		</React.Fragment>
+	);
+	const LogoutButton = (
+		<React.Fragment>
+			<ButtonDisplay key={2}  onClick={logout} buttonName="Logout" cssClass="seButton pull-right" />
+		</React.Fragment>
+	);
```

Comment:
nice, I like this home button. I think we can use this in some of the other forms as well

Reply:
these buttons will probably move up in the app, so that we don't have to include them in a bunch of different views, and are rather toggle their visibility. This is a nice step 1 though.